data-raw/healthy-places/access-to-green-space/access-greenspace.R

```diff
@@ -0,0 +1,30 @@
+# Load packages
+library(tidyverse)
+library(geographr)
+
+# Load lookup dataset
+lookup_ltla_raw <- lookup_ltla_ltla
+
+# Filter the dataset for Welsh local authorities - LTLA code and LTLA name
+raw_welsh_greenspace <- data.frame ()
+raw_welsh_greenspace <- lookup_ltla_raw |>
+  filter(startsWith(ltla21_code, "W0"))|>
+  select(ltla21_code, ltla21_name)
+print (raw_welsh_greenspace)
+
+# Add new columns for the green space data
+raw_welsh_greenspace$PPP <- NA
```

Comment:
- Column names should make sense to the reader, similar to object names. For example now I do not know what the columns PPP or N10 denotes. Please rename. Please also use snake case.
- Are all columns required, for example the column `NUTS1`? What is it added to the dataset?